GameServerLib/Items/ItemData.cs

```diff
@@ -62,6 +63,7 @@ public static ItemData Load(ItemManager owner, ItemContentCollectionEntry itemIn
                 Price = itemInfo.GetInt("Data", "Price"),
                 ItemGroup = itemInfo.GetString("Data", "ItemGroup"),
                 SpellName = itemInfo.GetString("Data", "SpellName"),
+                SpellData = new SpellData(game),
```

Comment:
Please change the coupling here if you have time. From what I see, the only use in `SpellData` is for the loading:
`_game.Config.ContentManager.GetContentFileFromJson("Spells", name);`

Change this to `ContentManager` or something less general than `Game` to discourage some workarounds using game's variables.

Reply:
I'll have to remove SpellData from ItemData if I wanted to do that since ContentManager isn't loaded it until after ItemData.